### PR TITLE
DOC: Add example to amax docstring

### PR DESCRIPTION
**Pull request by ankur0904**
Added example to the `amax` function of `fromnumeric.py`. #22266
@InessaPawson Please have a look.

### CODE REVIEW

File: numpy/core/fromnumeric.py [hunk 1]
```diff
@@ -2820,6 +2820,18 @@ def amax(a, axis=None, out=None, keepdims=np._NoValue, initial=np._NoValue,
     --------
     max : alias of this function
     ndarray.max : equivalent method
+
+    Examples
+    --------
+    >>> np.amax([4])
+    4
+    >>> np.amax([4,-1, 67, 89])
```
Comment (eendebakpt) [inline numpy/core/fromnumeric.py:2828]:
```suggestion
    >>> np.amax([4, -1, 67, 89])
```

File: numpy/core/fromnumeric.py [hunk 2]
```diff
@@ -2820,6 +2820,18 @@ def amax(a, axis=None, out=None, keepdims=np._NoValue, initial=np._NoValue,
...
+    89
+    >>> np.amax([[1,2,3,46,44],[2,3,4,55,5]], axis=0)
```
Comment (eendebakpt) [inline numpy/core/fromnumeric.py:2830]:
```suggestion
    >>> np.amax([[1, 2, 3, 46, 44], [2, 3, 4, 55, 5]], axis=0)
```
I prefer the example to follow pep8. (not sure this is required, but the example at https://numpydoc.readthedocs.io/en/latest/format.html#examples follows the style as well)

**Comment by ankur0904** [comment]
@eendebakpt I have made the required changes.

File: numpy/core/fromnumeric.py [hunk 3]
```diff
@@ -2820,6 +2820,18 @@ def amax(a, axis=None, out=None, keepdims=np._NoValue, initial=np._NoValue,
...
+
+    Examples
+    --------
+    >>> np.amax([4])
+    4
+    >>> np.amax([4, -1, 67, 89])
+    89
+    >>> np.amax([[1, 2, 3, 46, 44],[2, 3, 4, 55, 5]], axis=0)
+    array([2, 3, 4, 55, 44])
+    >>> np.amax([[1, 2, 3, 46, 44],[2, 3, 4, 55, 5]], axis=1)
```
Comment (eendebakpt) [inline numpy/core/fromnumeric.py:2832]:
```suggestion
    >>> np.amax([[1, 2, 3, 46, 44], [2, 3, 4, 55, 5]], axis=1)
```

**Review by eendebakpt — APPROVED** [review]

**Comment by ankur0904** [comment]
@eendebakpt I made the suggested change.

**Review by rossbar — COMMENTED** [review]
The docstring makes clear that `amax` is an alias for `max`. I'd vote to continue referring users to the `np.max` documentation rather than add explicit examples here.

**Comment by ankur0904** [comment]
@rossbar Alright.

**Comment by rossbar** [comment]
Thanks @ankur0904 , I'll go ahead and close this.

**Pull request closed without merging**